UITests/ClearPrivateDataTests.swift

```diff
@@ -13,7 +13,12 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         webRoot = SimplePageServer.start()
     }
 
-    override func tearDown() {
+    override func afterEach() {
+        BrowserUtils.resetToAboutHome(tester())
+    }
+
+
+    override func beforeAll() {
```

Comment:
Mostly because if there was a problem within ClearPrivateData itself leaving the test in a weird state (tearDown sometimes doesn't happen correctly after an error), then all other ClearPrivateData tests will fail. This is ensuring that within this test suite then whatever state each tests ends in, all subsequent tests will operate successfully.